subworkflows/local/annotate_genome_snvs.nf

```diff
@@ -28,7 +28,8 @@ workflow ANNOTATE_GENOME_SNVS {
         analysis_type         // string: [mandatory] 'wgs' or 'wes'
         ch_cadd_header        // channel: [mandatory] [ path(txt) ]
         ch_cadd_resources     // channel: [mandatory] [ path(annotation) ]
-        ch_vcfanno_resources  // channel: [mandatory] [ path(resources) ]
+        ch_vcfanno_extra      // channel: [mandatory] [ [path(vcf),path(index)] ]
+        ch_vcfanno_resources  // channel: [mandatory] [ [path(vcf),path(index),...] ]
```

Comment:
What does `...` mean?

Reply:
That it contains more than one vcf and index 😅

Reply:
What do you think about `[ path(vcfs), path(indices) ]`? 😁

Reply:
It's still one list of multiple vcfs `[ meta, [ vcf1, vcf2, vcf ], [tbi1, tbi2, tbi3] ]` right?  and not `[ meta, vcf1, tbi1, vc2, tbi2, etc... ]`

Reply:
I never put much thought into it.. But how about [path(vcf1),path(index1),...,path(vcfn),path(indexn)] instead)?

Reply:
Nope.. Its just a list containing paths and indices.. They are not grouped in any way :)

Reply:
Maybe I misunderstood, I edited my comment above.

Reply:
Ah, okay. then I guess just  `...` is fine!